wasi/src/test/java/com/dylibso/chicory/wasi/WasiPreview1Test.java

```diff
@@ -80,6 +82,82 @@ public void shouldRunWasiDemoJavyModule() {
         assertEquals(fakeStdout.output(), "{\"foo\":3,\"newBar\":\"baz!\"}");
     }
 
+    @Test
+    public void shouldUseQuickJsProvider() {
```

Comment:
Not really relevant to this PR, but it's a good reference for the future on how to programmatically use the provider QuickJS module.